frontend/frontend/src/components/ConfigBar/ConfigBar.vue

```diff
@@ -26,7 +26,7 @@
       </div>
       <b-card-body style="min-height: 71.4vh">
         <!--Render all param sliders with correct labels, for each tab-->
-        <div class="param-sliders" v-for="(tabData, index1) in tabsData" :key="tabData.data.label">
+        <div v-for="(tabData, index1) in tabsData" :key="tabData.data.label">
```

Comment:
```suggestion
        <div v-for="(tabData, index1) in tabsData" :key="index1">
```

You could just use the index as the unique key

Reply:
Apparently it's bad practise to use an index as a unique key as it can lead to issues if the iterable is later modified